pyproject.toml

```
@@ -16,7 +15,7 @@ requires-python = ">=3.9"
 
 description = "Foreign Function Interface for Python calling C code."
 readme = {file = "README.md", content-type = "text/markdown"}
-license = { file = "LICENSE" }
+license = "MIT"
```

Shouldn't this be [**MIT-0**](https://spdx.org/licenses/MIT-0.html) instead?
```suggestion
license = "MIT-0"
license-files = ["LICENSE"]
```

Adding license-files as the default glob in setuptools appears to be a left-over from early versions of PEP 639, see [Appendix: Rejected Ideas](https://peps.python.org/pep-0639/appendix-rejected-ideas/#use-a-default-value-for-license-files-if-not-specified):
> ### Use a default value for `license-files` if not specified
> 
> A previous draft of the PEP proposed a default value for detecting license files in case the users have not declared any and not marked the key as dynamic. That value was defined as an array of globs:
> `["LICEN[CS]E*", "COPYING*", "NOTICE*", "AUTHORS*"]`
> 
> However, this would create an exception among the existing metadata, as no other key has got implicit defaults defined. Implicit values in pyproject.toml keys are delegated to the `dynamic` field, which is specified as being calculated. Also, the values were chosen arbitrarily, without a strong justification why they should pose a standard.